tiagosatur/bluejack
# FEATURE | Create new pile page

**What does this PR do?**

It creates the new pile page and redirects to the next route

**How to test?**
- Run the project
- Click on "Start a new deck" button
- Click on the submit button
- You should be redirected to a route with the deck id
